<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorOscOutputWidget</class>
 <widget class="QWidget" name="InspectorOscOutputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>207</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutOutput">
     <item>
      <spacer name="horizontalSpacerOutput">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>26</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelOutput">
       <property name="text">
        <string>Output</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxOutput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutPath">
     <item>
      <spacer name="horizontalSpacerPath">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelPath">
       <property name="text">
        <string>Path</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPath"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutMessage">
     <item>
      <spacer name="horizontalSpacerMessage">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelMessage">
       <property name="text">
        <string>Message</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditMessage"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutType">
     <item>
      <spacer name="horizontalSpacerType">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>39</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelType">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Boolean</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Double</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Float</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Integer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>String</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacerOscOutput">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditPath</sender>
   <signal>textChanged(QString)</signal>
   <receiver>InspectorOscOutputWidget</receiver>
   <slot>pathChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditMessage</sender>
   <signal>textChanged(QString)</signal>
   <receiver>InspectorOscOutputWidget</receiver>
   <slot>messageChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxOutput</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>InspectorOscOutputWidget</receiver>
   <slot>outputChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxType</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>InspectorOscOutputWidget</receiver>
   <slot>typeChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pathChanged(QString)</slot>
  <slot>messageChanged(QString)</slot>
  <slot>outputChanged(QString)</slot>
  <slot>typeChanged(QString)</slot>
 </slots>
</ui>
